jupyter_releaser/npm.py

```diff
@@ -149,7 +149,7 @@ def get_package_versions(version):
         message += f'\nnpm version: {data["name"]}: {npm_version}'
     if "workspaces" in data:
         message += "\nnpm workspace versions:"
-        for path in _get_workspace_packages(data):
+        for path in sorted(_get_workspace_packages(data)):
```

Comment:
Maybe we should sort the result returned by that function so it is always sorted?

Reply:
(in the function itself)

Reply:
Yes sounds like it would be more convenient.